<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestAlveolesLibres</class>
 <widget class="QWidget" name="TestAlveolesLibres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestAlveolesLibres</string>
  </property>
  <widget class="QLabel" name="labelNumero">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro d'alvéoles libres</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetNumero">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRangees">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rangées &amp; colonnes occupées</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetRangees">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>82</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonReserver">
      <property name="text">
       <string>Réserver &gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLiberer">
      <property name="text">
       <string>&lt;&lt; Libérer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
